lib/dstu2_resources.rb

```diff
@@ -1458,132 +1458,289 @@ module Resources
       ]
     }
 
-    DSTU2_PATIENT_1 ||= { 
-      "resourceType" => "Patient",
-      "id" => "20940091",
-      "text" => {
-        "status" => "generated",
-        "div" =>
-          "&lt;div&gt;&lt;p&gt;&lt;b&gt;Patient&lt;/b&gt;&lt;/p&gt;&lt;p&gt;&lt;b&gt;Name&lt;/b&gt;: Pond, Melody&lt;/p&gt;&lt;p&gt;&lt;b&gt;DOB&lt;/b&gt;: 2012-04-25&lt;/p&gt;&lt;p&gt;&lt;b&gt;Sex&lt;/b&gt;: female&lt;/p&gt;&lt;p&gt;&lt;b&gt;Status&lt;/b&gt;: Active&lt;/p&gt;&lt;/div&gt;"
+    DSTU2_PATIENT_1 ||= {
+      "resourceType": "Patient",
+      "id": "1316037",
```

Comment:
Since this info will be publicly visible, can you tweak the ids and especially the names so that they don't identify real people?

To date we've had a Doctor Who theme for our fake people: Amy Pond, Melody Pond, River Song, Rory Williams